src/lib/radio/radio.spec.ts

```diff
@@ -773,6 +778,43 @@ describe('MatRadio', () => {
   });
 });
 
+describe('MatRadioDefaultOverrides', () => {
+  describe('when MAT_RADIO_DEFAULT_OPTIONS overridden', () => {
+    beforeEach(async(() => {
+      TestBed.configureTestingModule({
+        imports: [MatRadioModule, FormsModule],
+        declarations: [DefaultRadioButton, RadioButtonWithColorBinding],
+        providers: [{
+          provide: MAT_RADIO_DEFAULT_OPTIONS,
+          useValue: { color: 'color-provider-override' },
+        }],
+      });
+
+      TestBed.compileComponents();
+    }));
+    it('should override default color in Component', () => {
+      const fixture: ComponentFixture<DefaultRadioButton> =
+        TestBed.createComponent(DefaultRadioButton);
+      fixture.detectChanges();
+      const radioDebugElement: DebugElement =
+        fixture.debugElement.query(By.directive(MatRadioButton));
+      expect(
+        radioDebugElement.nativeElement.classList.contains('mat-color-provider-override')
+      ).toBeTruthy();
```

Comment:
You can reduce this to `expect(radioDebugElement.nativeElement.classList).toContain('some-class')`. Same goes for all the other assertions like this one.

Also in general prefer to use valid values in unit tests, rather than made up ones. `color-provider-override` is invalid.